src/Templates/src/templates/maui-mobile/MauiApp1/MauiApp1.csproj

```diff
@@ -3,7 +3,11 @@
 	<PropertyGroup>
 		<!-- iOS, Android, MacCatalyst -->
 		<TargetFrameworks>net6.0-ios;net6.0-android;net6.0-maccatalyst</TargetFrameworks>
+		<!-- Windows -->
+		<TargetFrameworks Condition="$(OS.Contains('Windows'))">net6.0-windows10.0.19041;$(TargetFrameworks)</TargetFrameworks>
```

Comment:
The usual `'$(OS)' == 'Windows_NT'` seems to get removed when the template is packaged. Need to look at that, but this is also good.